mars/services/task/supervisor/processor.py

```diff
@@ -65,11 +67,15 @@ def __init__(
             ProfilingData.init(task.task_id, task.extra_config["enable_profiling"])
 
         self._dump_subtask_graph = False
+        self._collect_info = False
```

Comment:
1. self._collect_info -> self._collect_task_info

2. assign the value directly:
`self._collect_info = task.extra_config and task.extra_config.get("collect_info", False)`